src/global/conditionals/simple_programs_with_decision_points/medium/q2/NumberJumble.java

```diff
@@ -0,0 +1,22 @@
+package global.conditionals.simple_programs_with_decision_points.medium.q2;
+import java.util.Scanner;
+
+/*
+Write a program that asks the user for a number and divides the number by 10 (using integer division) if it is greater
+than 10, and multiplies it by 10 if it is less than 10.
+ */
```

Comment:
I know the program has the if statement being >10 and then an else, but the question disregards the number 10. It is either greater than or less than, so maybe an update to less than or equal to 10?